### PR TITLE
Fix/issue 143

### PR DESCRIPTION
**Pull request by VivekFitkariwala**
The PR contains fixes for
- Taking types directly from the types folder instead of having a dependency in package.json
- Updating boilerplate of frontend, backend and clientLib to remove upgrade_types as a dependency.
- Updating github actions accordingly

### CODE REVIEW

File: types/package.json [hunk 1]
```diff
@@ -2,11 +2,8 @@
   "name": "upgrade_types",
   "version": "1.0.5",
   "description": "",
-  "main": "dist/index.js",
-  "types": "dist/index.d.ts",
-  "files": [
-    "dist/*"
-  ],
+  "main": "src/index.ts",
+  "types": "src/index.ts",
```
Comment (ppratikcr7) [inline types/package.json:6]: Just to confirm, types would also be using the same `index.ts` file and not the `index.d.ts` as earlier?

**Review by ppratikcr7 — APPROVED** [review]
Just check for merge conflicts, rest looks fine.

File: types/package.json (hunk 1 above)
Comment (VivekFitkariwala) [reply types/package.json:6]: Yup now removing the build process from the types and consuming typescript files directly by other projects.

Comment (ppratikcr7) [reply types/package.json:6]: okay got it.

**Pull request merged**